connectors/byoc.py

```diff
@@ -237,6 +237,12 @@ async def done(self, indexed_count=0, deleted_count=0, exception=None):
 
         return await self.client.update(index=JOBS_INDEX, id=self.job_id, doc=job_def)
 
+    async def suspend(self):
+        self.status = JobStatus.SUSPENDED
+        job_def = {"status": e2str(self.status)}
```

Comment:
I'm surprised to see `suspended_at` is not in [the protocol](https://github.com/elastic/connectors-ruby/blob/main/docs/CONNECTOR_PROTOCOL.md). But looking at the [Monitoring project brief](https://docs.google.com/document/d/1mqPTc3Agg7FaUPVZ0YR4_R3M552HjqFwAL1UJvi57Ek/edit), it seems this is not required:

> timestamps when
> - job requested/pending
> - job started
> - job completed
> - job canceling
> - job canceled / aborted



Hi @danajuratoni , do we need the timestamp when the job is suspended?

Reply:
True, we haven't added it yet! Waiting for Dana's response.